### ISSUE TITLE
presync hook fails with panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by andrewnazarov** [roboll/helmfile#1459]
`helmfile apply` fails to install releases with the following error:

```
processing releases in group 2/4: cert-manager/cert-manager, monitoring/prom, logging/efklogs
helmfile.yaml: basePath=.
hook[kubectl]: triggered by event "presync"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x767222]
goroutine 187 [running]:
go.uber.org/zap.(*SugaredLogger).log(0x0, 0xff, 0x15ff253, 0x4, 0xc000483e20, 0x2, 0x2, 0x0, 0x0, 0x0)
	/home/circleci/workspace/helmfile/vendor/go.uber.org/zap/sugar.go:221 +0x192
go.uber.org/zap.(*SugaredLogger).Debugf(...)
	/home/circleci/workspace/helmfile/vendor/go.uber.org/zap/sugar.go:133
github.com/roboll/helmfile/pkg/helmexec.(*logWriter).Write(0xc000725c80, 0xc000634000, 0x5c, 0x8000, 0x5c, 0x0, 0x0)
	/home/circleci/workspace/helmfile/pkg/helmexec/log.go:25 +0x16c
io.(*multiWriter).Write(0xc000725ca0, 0xc000634000, 0x5c, 0x8000, 0x5c, 0x0, 0x0)
	/usr/local/go/src/io/multi.go:60 +0x87
io.copyBuffer(0x1b66740, 0xc000725ca0, 0x1b66c00, 0xc0006140b8, 0xc000634000, 0x8000, 0x8000, 0x405125, 0xc0005243c0, 0xc00028ffb0)
	/usr/local/go/src/io/io.go:407 +0x1fb
io.Copy(...)
	/usr/local/go/src/io/io.go:364
os/exec.(*Cmd).writerDescriptor.func1(0xc0005243c0, 0xc00028ffb0)
	/usr/local/go/src/os/exec/exec.go:311 +0x63
os/exec.(*Cmd).Start.func1(0xc0004882c0, 0xc000725d20)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x632
```

I've got cert-manager defined in helmfile manifest with a presync hook:
```
- name: cert-manager
    namespace: cert-manager
    chart: jetstack/cert-manager
    version: v0.12.0
    needs:
      - ingress/nginx-ingress
    labels:
      tier: common
      type: ingress
    hooks:
      - events: ["presync"]
        command: "kubectl"
        args: ["apply", "--validate=false", "-f", "https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/deploy/manifests/00-crds.yaml"]
```

versions:
```
HELM_VERSION=v3.2.4
HELMFILE_VERSION=v0.126.0
HELM_DIFF_VERSION=v3.1.3
```

**Comment by mumoshu** [comment]
@andrewnazarov Thanks for reporting! I believe this has been fixed since v0.126.1 by https://github.com/roboll/helmfile/pull/1448

**Comment by andrewnazarov** [comment]
Yes, the latest version works. Thank you!
